vertical-pod-autoscaler/README.md

```diff
@@ -48,12 +48,13 @@ procedure described below.
 
 # Installation
 
-The current default version is Vertical Pod Autoscaler 0.13.0
+The current default version is Vertical Pod Autoscaler 0.14.0
 
 ### Compatibility
 
 | VPA version     | Kubernetes version |
 |-----------------|--------------------|
+| 0.14            | 1.25+              |
```

Comment:
Did we test with 1.25?

Reply:
Yes, following the procedure directly meant that I used GKE cluster with version 1.25.8-gke.1000 (both KCP & nodes).